venom_core/services/model_introspection_activation_path_service.py

```
@@ -220,6 +224,94 @@ def _build_tensor_activation_contract(
         )
         cosine_similarity = round(dot_product / (mlp_norm * residual_norm), 6)
 
+    from venom_core.services.model_introspection_operator_trends_service import (
+        get_operator_run_records,
+    )
+
+    try:
+        past_runs = get_operator_run_records(settings=settings)
+    except Exception:
```

Ruff (0.15.14)

[warning] Do not catch blind exception: `Exception`

(BLE001)